tests/test_raster.py

```diff
@@ -987,10 +1023,6 @@ def test_crop(self, data: list[str]) -> None:
         r_cropped = r.crop(crop_geom2)
         assert r_cropped.raster_equal(r)
 
-        # Test with bracket call
-        r_cropped_getitem = r[crop_geom2]
```

Comment:
so this functionality is deleted now?

Reply:
Yes! It was not in the documentation yet, and was not used much...

Reply:
The problem was that NumPy indexes can overlap with `crop_geom` types: If you pass [0, 1, 2, 3], you can't know if you are supposed to crop between [0, 2] and [1, 3], or index those 4 values?
I had repeated feedback from users that it would be more practical to be able to do raster[index] instead of raster.data[index] to manipulate the array.

Reply:
Good point ! I'm convinced now 😉